src/utils/src/url.js

```diff
@@ -9,6 +9,7 @@ import { isDefined } from './object';
 
 /**
  * @private
+ * Adapted from https://github.com/lakenen/node-append-query, to fit a non-node runtime.
  */
 function serialize(obj, options = {}, prefix) {
```

Comment:
Good question with the `qs` library. I was also concerned about using multiple utils to encode, but @thomasconner might know the reasoning behind it.